### PR TITLE
Generate demodata bugfix

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

**Comment by xliiv** [comment]
Don't we generate this data anymore?

**Comment by ghost** [comment]
:ok:

**Comment by mkurek** [comment]
Couldn't we generate demo data in a single test?

File: src/ralph/data_importer/management/commands/demodata.py [hunk 1]
```diff
@@ -217,703 +221,699 @@ def add_user_and_group(username, password, group_name, permissions):
             user.save()
 
         dc_permissions = [
```
Comment (mkurek) [inline src/ralph/data_importer/management/commands/demodata.py]: do we have to list every possible permission? couldn't we just grant all in particular app or app.model?

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**